README.md

```diff
@@ -23,6 +23,8 @@ This project is currently in development! It will be announced when there is a (
   S3_BUCKET=<your-s3-bucket>
   GITHUB_ID=<your-github-client-id>
   GITHUB_SECRET=<your-github-client-secret>
+  GOOGLE_ID=<your-google-client-id> (use google+ api)
```

Comment:
These are different from what's in `server/config/passport.js`. I think these variables should be called `GOOGLE_ID` and `GOOGLE_SECRET`, so I would change the other file.

Also, these variables need to be added to the "Production Setup" .env file example in this README.

Reply:
@catarak 
Oops! my bad. I changed it. 
Thanks.